.circleci/config.yml

```diff
@@ -848,7 +848,7 @@ jobs:
           command: go tool dist list | grep mips
       - run:
           name: Pull Espresso Docker images
-          command: make e2e-pull
+          command: make op-e2e e2e-pull
```

Comment:
What does this do? I don't see an `op-e2e` target in the `Makefile`

Reply:
yeah didn't mean to commit this